emily/handler/src/api/handlers/deposit.rs

```diff
@@ -343,17 +343,9 @@ pub async fn create_deposit(
             },
         )
         .await;
-        // Reject if we already have a deposit with the same txid and output index and it is NOT pending or reprocessing.
+
         match entry {
-            Ok(deposit) => {
-                if deposit.status != Status::Pending && deposit.status != Status::Reprocessing {
-                    return Err(Error::Conflict);
-                } else {
-                    // If the deposit is pending or reprocessing, we should keep height and hash same as in the old deposit
-                    stacks_block_hash = deposit.last_update_block_hash;
-                    stacks_block_height = deposit.last_update_height;
-                }
-            }
+            Ok(_) => return Err(Error::Conflict),
```

Comment:
After thinking about this some more, I think that we should return `200 OK` here instead of an error. Well, we haven't validated the request yet so maybe we should do that before reaching out to the database. If the body passes validation then then we can reach out the database and return `200 OK` is it exists.